src/block/tab-labels/block.json

```
@@ -0,0 +1,15 @@
+{
+	"apiVersion": 2,
+	"name": "stackable/tab-labels",
+	"title": "Tab Labels",
+	"description": "Container for Tabs Block labels",
+	"category": "stackable",
+	"usesContext": [ "postId", "postType", "queryId" ],
+	"parent": [
+		"stackable/tabs"
+	],
+	"keywords": [],
+	"textdomain": "stackable-ultimate-gutenberg-blocks",
+	"stk-type": "special",
```

You'll need to set this and Tab Content to `hidden` so that the block won't show up in the enable/disable block are in the settings
```suggestion
	"stk-type": "hidden",
```